EIPS/eip-FOCIL.md

```diff
@@ -16,28 +16,73 @@ Implement a robust mechanism to preserve Ethereum’s censorship resistance and
-- Attesters only vote for the proposer's block if it includes transactions from local inclusion lists they collected.
+- Attesters only vote for the proposer's block if it includes transactions from local ILs they collected.
 
 ## Motivation
 
 In an effort to shield the Ethereum validator set from centralizing forces, the right to build blocks has been auctioned off to specialized entities known as builders. Over the past year, this has resulted in a few sophisticated builders dominating the network’s block production. Economies of scale have further entrenched their position, making it increasingly difficult for new entrants to gain significant market share. A direct consequence of centralized block production is a deterioration of the network’s censorship resistance properties. In contrast, 90% of the more decentralized and heterogeneous validator set is not engaging in censorship. This has driven research toward ways that allow validators to impose constraints on block construction by force-including transactions in the blocks. These efforts recently culminated in the first practical implementation of forward ILs, [EIP-7547](./eip-7547.md), being considered for inclusion in the upcoming Pectra fork. However, some concerns were raised about the specific mechanism proposed in EIP-7547, leading to its rejection. 
 
-FOCIL is a simple committee-based design improving upon previous IL mechanisms or block co-creation proposals and addressing issues related to bribing/extortion attacks, IL equivocation, account abstraction (AA), and incentive incompatibilities.
+FOCIL is a simple committee-based design improving upon previous IL mechanisms or block co-creation proposals and addressing issues related to bribing/extortion attacks, IL equivocation, account abstraction (AA) and transaction invalididation.
+
+## High-level Overview
+
+### Roles And Participants
+
+This section outlines the workflow of FOCIL, detailing the roles and responsibilities of various participants, including IL committee members, nodes, proposers, and attesters.
+
+##### IL Committee Members
+
+- **`Slot N`, `t=0 to 8s`**:
+IL committee members construct their local ILs and broadcast them over the P2P network after processing the block for `slot N` and confirming it as the head. If no block is received by `t=7s`, they should run `get_head` and build and release their local ILs based on their node’s canonical head.
+
+  By default, local ILs are built by selecting raw transactions from the public mempool, ordered by priority fees, up to the local IL’s maximum size in bits (e.g., 8 KB per local IL). Additional local rules can be optionally applied to maximize censorship resistance, such as prioritizing valid transactions that have been pending in the mempool the longest.
+
+##### Nodes
+
+- **`Slot N`, `t=0 to 9s`**:
+Nodes receive local ILs from the P2P network and only forward and cache those that pass the CL P2P validation rules.
```

Comment:
If the rule is indeed as simple as "only forward up to two ILs per IL proposer", I think it could make sense to specify it here?